src/common/AllExperiments/Experiments_Inert.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace AllExperiments
+{
+#pragma warning disable SA1649 // File name should match first type name.
```

Comment:
We suppressed this warning because this class needs to be called Experiments, so its the same name as the class in Experiments_Live. Is there a "nicer" way to suppress this warning?

Reply:
This warning is about the name of the file matching the class within. This is something we want on for code quality. This is an exception that makes sense for this specific case. I think having a local disable like yo're doing is likely the best alternative here. Please add a comment about why this exception is needed? Thank you!